adafruit_pyportal.py

```diff
@@ -399,6 +407,9 @@ def __init__(
                 self._text_wrap[i] = text_wrap[i]
                 self._text_maxlen[i] = text_maxlen[i]
                 self._text_transform[i] = text_transform[i]
+                if not isinstance(text_scale[i], (int, float)) or text_scale[i] < 1:
```

Comment:
I tested these changes and I am getting and I'm getting an exception from this line:
```
Traceback (most recent call last):
  File "code.py", line 80, in <module>
  File "/lib/adafruit_pyportal.py", line 410, in __init__
TypeError: 'int' object is not subscriptable
```
I'm using the code from the quotes example here: https://learn.adafruit.com/adafruit-pyportal/parsing-json. I tried with the code as-is, and with the new `text_scale` parameter added, I'm getting same results both ways.

Reply:
I was able to get past this error by adding a check for `text_scale` type check inside the body of the if statement at line 368 of the modified code. Now mine looks like:
```python
if isinstance(text_position[0], (list, tuple)):
    num = len(text_position)
    if not text_wrap:
        text_wrap = [0] * num
    if not text_maxlen:
        text_maxlen = [0] * num
    if not text_transform:
        text_transform = [None] * num
    if isinstance(text_scale, int):  # new check for text scale
        text_scale = [text_scale] * num
```